### DISCUSSION TITLE
[Unstable] Additional skill gain once every round

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#14737] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If the player character has any bonus skill it will affect regular skill gain once per round.
For instance, a captain with the bonus +10 Helm from the hat will receive +10 Helm skill from piloting the sub instead of just +1.
Another example using The Sailor's Guide book with Picking Up the Slack and wearing a Researcher's Outfit:

https://github.com/user-attachments/assets/ab3b1aba-6a5d-4fea-b0d0-a6058b9241db

In this case the bonus skill gets added to the book's skill gain.

### Reproduction steps

1. Start a multiplayer mission/campaign
2. Use The Sailor's Guide book while wearing skill boosting clothes

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

Other

### -

v1.6.9.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this, no issues found.
Closing discussion as resolved.
